src/main/scala/se/lu/nateko/cp/data/formats/dailysitescsv/DailySitesCsvStreams.scala

```diff
@@ -0,0 +1,72 @@
+package se.lu.nateko.cp.data.formats.dailysitescsv
+
+import java.time.LocalTime
+
+import DailySitesCsvParser.{Header, parseLine, seed}
+import akka.stream.scaladsl.{Flow, Framing, Keep, Sink}
+import akka.util.ByteString
+import akka.{Done, NotUsed}
+import se.lu.nateko.cp.data.formats.bintable.BinTableRow
+import se.lu.nateko.cp.data.formats.{ColumnFormats, ParsingAccumulator, ProperTableRow, ProperTableRowHeader}
+import se.lu.nateko.cp.meta.core.data.{TimeInterval, TimeSeriesUploadCompletion}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class DailySitesCsvRow(val header: Header, val cells: Array[String]) extends ProperTableRow[Header]
+
+object DailySitesCsvStreams {
+	implicit class TimeSeriesParserEnhancer[A <: ParsingAccumulator, M](val parser: Flow[String, A, M]) extends AnyVal {
```

Comment:
Does not seem to be DailySitesCsv-specific. Should it not be reused from elsewhere?